### PR TITLE
GIX-1247: Add SNS Aggregator mainnet in config

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Allow mainnet nns-dapp to integrate with SNS Aggregator.

# Changes

* Add mainnet data for SNS aggregator in `dfx.json`.

# Tests

* Not applicable

### CODE REVIEW

**Comment by lmuntaner** [comment]
Mainnet config:

```
➜  nns-dapp git:(GIX-1247_LM_add-sns-aggregator-mainnet) ✗ DFX_NETWORK=mainnet ./config.sh
~/Documents/nns-dapp ~/Documents/nns-dapp
VITE_DFX_NETWORK=mainnet
VITE_CYCLES_MINTING_CANISTER_ID=rkp4c-7iaaa-aaaaa-aaaca-cai
VITE_WASM_CANISTER_ID=qaa6y-5yaaa-aaaaa-aaafa-cai
VITE_GOVERNANCE_CANISTER_ID=rrkah-fqaaa-aaaaa-aaaaq-cai
VITE_GOVERNANCE_CANISTER_URL=https://rrkah-fqaaa-aaaaa-aaaaq-cai.icp-api.io
VITE_LEDGER_CANISTER_ID=ryjl3-tyaaa-aaaaa-aaaba-cai
VITE_LEDGER_CANISTER_URL=https://ryjl3-tyaaa-aaaaa-aaaba-cai.icp-api.io
VITE_OWN_CANISTER_ID=qoctq-giaaa-aaaaa-aaaea-cai
VITE_OWN_CANISTER_URL=https://nns.internetcomputer.org
VITE_FETCH_ROOT_KEY=false
VITE_FEATURE_FLAGS="{\"ENABLE_CKBTC_LEDGER\":true,\"ENABLE_CKBTC_RECEIVE\":false,\"ENABLE_SNS_2\":false,\"ENABLE_SNS_AGGREGATOR\":false,\"ENABLE_SNS_VOTING\":false}"
VITE_HOST=https://icp-api.io
VITE_IDENTITY_SERVICE_URL=https://identity.internetcomputer.org/
VITE_AGGREGATOR_CANISTER_URL=https://3r4gx-wqaaa-aaaaq-aaaia-cai.icp0.io
VITE_CKBTC_LEDGER_CANISTER_ID=mxzaz-hqaaa-aaaar-qaada-cai
VITE_CKBTC_INDEX_CANISTER_ID=n5wcd-faaaa-aaaar-qaaea-cai
Config has been defined in '/Users/llorencmuntaner/Documents/nns-dapp/frontend/.env'
~/Documents/nns-dapp
FIN
```

VITE_AGGREGATOR_CANISTER_URL works as expected. You can visit it here: https://3r4gx-wqaaa-aaaaq-aaaia-cai.icp0.io/

**Comment by lmuntaner** [comment]
@dskloetd @bitdivine please review

File: dfx.json [hunk 1]
```diff
@@ -223,7 +228,8 @@
     "mainnet": {
       "config": {
         "FETCH_ROOT_KEY": false,
-        "HOST": "https://icp-api.io",
+        "API_HOST": "https://icp-api.io",
+        "STATIC_HOST": "https://icp0.io",
```
Comment (dskloetd) [inline dfx.json:232]: Usage of the *_HOST variables looks correct but I don't know about the URLs or the dfx.json changes.

Just adding this comment so my approval alone won't trigger automerge.

**Review by dskloetd — APPROVED** [review]

File: dfx.json (hunk 1 above)
Comment (lmuntaner) [reply dfx.json:232]: Ok, thanks!

File: dfx.json [hunk 2]
```diff
@@ -190,7 +190,12 @@
       ],
       "candid": "rs/sns_aggregator/sns_aggregator.did",
       "wasm": "sns_aggregator.wasm",
-      "type": "custom"
+      "type": "custom",
+      "remote": {
+        "id": {
+          "mainnet": "3r4gx-wqaaa-aaaaq-aaaia-cai"
```
Comment (bitdivine) [inline dfx.json:196]: I can confirm that `3r4gx-wqaaa-aaaaq-aaaia-cai` is the aggregator canister ID on mainnet

**Review by bitdivine — APPROVED** [review]

File: dfx.json (hunk 2 above)
Comment (lmuntaner) [reply dfx.json:196]: thanks!

**Pull request merged**